and-bible/app/src/main/java/net/bible/android/control/link/LinkControl.java

```diff
@@ -297,6 +297,13 @@ else if(windowMode.equals(WINDOW_MODE_NEW)) {
 
 			windowControl.addNewWindow(document, key);
 		}
+		else if(windowMode.equals(WINDOW_MODE_THIS)) {
+			if (document==null) {
+				document = defaultDocument;
+			}
+			windowControl.getActiveWindow().setSynchronised(false);

```

Comment:
I realise this was suggested but switching off sync is potentially confusing because it is not obvious to the user why this has happened.  I think I preferred the previous issue of other windows being sync'd - they can always press Back.

Reply:
I agree that this is disputable. But in a way I found it something that might be often desired in typical use case.